src/phenCryo.ttl

```diff
@@ -41,33 +43,74 @@ sophcr:Calving rdf:type owl:Class ;
 ###  http://sweetontology.net/phenCryo/Englacial
 sophcr:Englacial rdf:type owl:Class ;
                 rdfs:subClassOf sophcr:GlacialProcess ;
-                rdfs:label "englacial"@en .
+                rdfs:label "englacial"@en ;
+                skos:closeMatch <http://purl.obolibrary.org/obo/ENVO_01001622>;
+                skos:definition  [
+                  rdfs:comment  "A positional quality inhering in a bearer by virtue of the bearer being located within a glacier or ice sheet, between their summer surface or bed."@en ;
+                  dcterms:source <https://orcid.org/0000-0003-4808-4736> ;
+                  dcterms:created "2020-04-09T10:12:52-08:00Z"^^xsd:dateTimeStamp ;
+                  dcterms:creator <https://orcid.org/0000-0003-4091-6059> ;
+                  prov:wasDerivedFrom <http://purl.obolibrary.org/obo/ENVO_01001622> ;
+                ] .
 
 
 ###  http://sweetontology.net/phenCryo/Glacial
 sophcr:Glacial rdf:type owl:Class ;
               owl:equivalentClass sophcr:Glaciation ;
-              rdfs:label "glacial"@en .
+              rdfs:label "glacial"@en ;
+              skos:definition  [
+                rdfs:comment  "An environmental process which involves glaciers or ice sheets."@en ;
```

Comment:
Again this is less an issue with the changes made to the file, than it is with the label SWEET gave the term.  This term is in the file phenCryo.ttl which is all about large scale processes; so all terms were addressed with that perspective.

In the Global Cryosphere Watch compilation of 27 cryospheric glossaries two distinct definitions of the term "glacial" exist - one from the Meteorology community the other from physical geography.  I am not aware of a Neuendor et al 2005 glossary and would need a full citation to find it.  The GCW definitions are:

(1) Period of time during an ice age when glaciers advance because of colder temperatures. 

(2) Involving glaciers and moving ice. Usually pertaining to processes associated with glaciers and ice sheets.

Clearly one is talking about a time period, the other about a process.  Since this term is in the SWEET phenomenon (i.e., large scale processes) section, the assumption is that the process version is the correct definition.

However, it also should be noted that the term Glaciation also has three definitions in the GCW compilation, the last of which is process related, thus the equivalence class.  Also, we did not add a mapping to ENVO because ENVO does not have a glaciation process term, just the time period term:

1. In meteorology, the term indicates that process of transformation of cloud particles from supercooled water drops to ice crystals has occurred. Thus, a cumulonimbus cloud is said to have a glaciated upper portion.

2. As used in many texts, particularly with respect to the ice ages, same as glacierization, for example, "Pleistocene glaciation."

3. When any part of the earth's surface has been altered by passage of a glacier, chiefly by glacial erosion or deposition; as distinguished from the term glacierization, indicating land currently being covered by glacier ice.

Reply:
Neuendorf, K.K.E., Mehl Jr, J.P., and Jackson, J.A., 2005, Glossary of Geology, Fifth Edition: Alexandria, VA, American Geological Institute, 779 pages.   I assumed anyone doing geology related stuff would be familiar with the AGI Glossary. Sorry!

Reply:
Collective comments from our [Semantic Harmonization call](
https://docs.google.com/document/d/1IFA6AsCdxESh9mZB5W1bZ_vARFkF3OvnL-KgdGNcp60/edit#heading=h.2uotl4egsd38)

See comments above and below linked to #185

Reply:
see #185, #195